pkg/network/netlink/consumer.go

```diff
@@ -337,6 +337,41 @@ func (c *Consumer) dumpTable(family uint8, output chan Event, ns netns.NsHandle)
 	})
 }
 
+// LoadNfConntrackKernelModule requests a dummy connection tuple from netlink conntrack which is discarded but has
+// the side effect of loading the nf_conntrack_netlink module
+func LoadNfConntrackKernelModule(ns netns.NsHandle) error {
+	sock, err := NewSocket(ns)
+	if err != nil {
+		return fmt.Errorf("could not open netlink socket for net ns %d: %w", int(ns), err)
+	}
+	defer sock.Close()
+
+	conn := netlink.NewConn(sock, sock.pid)
+	defer func() {
+		_ = conn.Close()
+	}()
+
+	dummyTupleData := []byte{0x2, 0, 0, 0, 0, 0, 0}
+
+	req := netlink.Message{
+		Header: netlink.Header{
+			Type:  netlink.HeaderType((unix.NFNL_SUBSYS_CTNETLINK << 8) | ipctnlMsgCtGet),
+			Flags: netlink.Request,
+		},
+		Data: dummyTupleData,
+	}
+
+	if _, err = conn.Send(req); err != nil {
+		log.Warnf("Error while sending netlink request: %v", err)
```

Comment:
nit: the other logs start with lower case, change this (and the one below) to be consistent.